src/server/game/Achievements/AchievementMgr.cpp

```diff
@@ -591,8 +591,8 @@ void AchievementMgr::LoadFromDB(PreparedQueryResult achievementResult, PreparedQ
 
             // title achievement rewards are retroactive
             if (AchievementReward const* reward = sAchievementMgr->GetAchievementReward(achievement))
-                if (uint32 titleId = reward->titleId[Player::TeamForRace(GetPlayer()->getRace()) == ALLIANCE ? 0 : 1])
```

Comment:
stooooooooop right here, `uint32 titleId` is a local variable scoped to this if only, it should not start with uppercase (this comment applies to ALL local variables changed in this PR)